packages/styles/stories/Components/card/object-card.example.html

```diff
@@ -19,38 +19,38 @@
             <div class="fd-card__content" role="group" aria-label="Card Content">
                 <div class="fd-card__content-container">
                     <div class="fd-card__content-section">
-                        <div class="fd-card__content-title-container">Group Title 1</div>
+                        <div class="fd-card__content-title-container"><h3>Group Title 1</h3></div>
                         <div class="fd-card__content-group">
                             <div class="fd-card__content-group-text">
-                                <span>Label</span>
-                                <span>Text</span>
+                                <span>Label:</span>
+                                <span class="fd-link">Text</span>
```

Comment:
you can't style a span as link. This should be reverted for the entire file.
What goes in this container is up to the user. We only provide the spacing for the container.